src/Components/Input/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { View, Text, TextInput } from 'react-native';
```

Comment:
O import do `Text` não tá sendo utilizado.
```suggestion
import { View, TextInput } from 'react-native';
```